quizapp/src/main/java/billennium/quizapp/entity/Answer.java

```diff
@@ -0,0 +1,34 @@
+package billennium.quizapp.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ManyToMany;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Answer extends BaseModel {
+
+    @NotNull(message = "No answer text provided.")
+    @Column(columnDefinition = "TEXT")
+    private String text;
+
+    @ManyToMany
```

Comment:
https://vladmihalcea.com/the-best-way-to-use-the-manytomany-annotation-with-jpa-and-hibernate/

Reply:
bez mapped by ?